pkg/cluster/install.go

```diff
@@ -93,6 +94,18 @@ func (m *manager) adminUpdate() []steps.Step {
 		)
 	}
 
+	if isRenewCerts {
```

Comment:
I think it would be preferable if we followed a similar format to the rest of the ordered list, and only have each action listed in one section.

So for https://github.com/Azure/ARO-RP/blob/0f950d78f14cc4391a90520f12958327482c077d/pkg/cluster/install.go#L70-L84, you'd do something like:
```
	// Requires Kubernetes clients
	if isEverything {
		toRun = append(toRun,
			steps.Action(m.fixSREKubeconfig),
			steps.Action(m.fixUserAdminKubeconfig),
			steps.Action(m.createOrUpdateRouterIPFromCluster),
			steps.Action(m.ensureGatewayUpgrade),
			steps.Action(m.populateRegistryStorageAccountName),
			steps.Action(m.ensureMTUSize),
		)
	}

	// Renew all certificate pieces
	if isEverything || isRenewCerts {
		toRun = append(toRun,
			steps.Action(m.fixMCSCert),
			steps.Action(m.fixMCSUserData),
			steps.Action(m.ensureGatewayUpgrade),
			steps.Action(m.configureAPIServerCertificate),
			steps.Action(m.configureIngressCertificate),
			steps.Action(m.renewMDSDCertificate),
		)
	}
```